### DISCUSSION TITLE
Plugins developement

### DISCUSSION THREAD

**Discussion by AVariot** [netbox-community/netbox#13374] [Help Wanted!]
Hi it's again me.
I'm working on a netbox plugin and i need to modified the device ObjectView.
![image](https://github.com/netbox-community/netbox/assets/91867982/aad5c13f-bd48-4713-9d89-833a4206b697)
Like on the photo i want to add a new tab like changelog but i don't know how to do it.
Someone has the response ?
Thank you for Your read time.

**Comment by candlerb** [comment]
https://docs.netbox.dev/en/stable/plugins/development/views/#extending-core-views

**Reply by AVariot** [reply]
Thank you for the help
